connector.go

```diff
@@ -43,6 +42,21 @@ func connectorConsumer[Type any, Status StatusType](w *Workflow[Type, Status], c
 		fmt.Sprintf("%v", totalShards),
 	)
 
+	errBackOff := w.defaultOpts.errBackOff
+	if config.errBackOff.Nanoseconds() != 0 {
+		errBackOff = config.errBackOff
+	}
+
+	lagAlert := w.defaultOpts.lagAlert
+	if config.lagAlert.Nanoseconds() != 0 {
+		lagAlert = config.lagAlert
+	}
+
+	lag := w.defaultOpts.lag
+	if config.lag.Nanoseconds() != 0 {
```

Comment:
A duration is just an int64, so I think you can do `config.lag > 0`. Or you can do `lag := cmp.Or(config.lag, w.defaultOpts.lag)`.

Reply:
Good point